vdev/src/commands/meta/compile_vrl_wasm.rs

```diff
@@ -0,0 +1,36 @@
+use std::path::Path;
+use anyhow::Result;
+use clap::Args;
+
+use crate::app;
+
+/// Compiles VRL crates to wasm32-unknown-unknown
+#[derive(Args, Debug)]
+#[command()]
+pub struct Cli {}
+
+impl Cli {
+    pub fn exec(self) -> Result<()> {
+        let vrl_path = Path::new(app::path()).join("lib").join("vrl");
+        // Can't use cargo vdev build because it defaults to Error: command: "cargo" "build" "--no-default-features" "--release" "--features" "default" "--target" "wasm32-unknown-unknown"
+        // Notice the --features "default", some of the crates below don't have default specified
```

Comment:
I'm not sure what the problem is here. The script was running `cargo build` not `vdev build`?

Reply:
Correct, the error I experienced was that after I changed directories and ran `cargo vdev build --release wasm32-unknown-unknown` vdev would then run with:  `"cargo" "build" "--no-default-features" "--release" "--features" "default" "--target" "wasm32-unknown-unknown"` which would fail because cargo wouldn't be able to find `default` feature list in I'm assuming the Cargo.toml for the directory that `cargo` command was ran in.